packages/http-server/src/getHttpConfigFromRequest.ts

```diff
@@ -6,11 +6,15 @@ import * as D from 'io-ts/lib/Decoder';
 import * as parsePreferHeader from 'parse-prefer-header';
 
 const BooleanFromString = D.parse<string, boolean>(s =>
-  s === 'true' ? D.success(true) : s === 'false' ? D.success(false) : D.failure(s, 'Not a boolean')
+  s === 'true' ? D.success(true) : s === 'false' ? D.success(false) : D.failure(s, 'a boolean')
+);
+
+const IntegerFromString = D.parse<string, number>(s =>
+  !isNaN(parseInt(s)) ? D.success(parseInt(s)) : D.failure(s, 'a number')
```

Comment:
```suggestion
  !isNaN(parseInt(s, 10)) ? D.success(parseInt(s, 10)) : D.failure(s, 'a number')
```

Reply:
Do not parse the integer twice. Save the result in a variable and reuse it both for the test and for the returned value.

Reply:
Done. c00bfcb